metadata-ingestion/src/datahub/ingestion/transformer/base_transformer.py

```diff
@@ -216,15 +247,7 @@ def transform(
                             else None,
                         )
                         if transformed_aspect:
-                            # for end of stream records, we modify the workunit-id
                             structured_urn = Urn.create_from_string(urn)
-                            simple_name = "-".join(structured_urn.get_entity_id())
-                            record_metadata = envelope.metadata.copy()
-                            record_metadata.update(
-                                {
-                                    "workunit_id": f"txform-{simple_name}-{self.aspect_name()}"
```

Comment:
actually I was wrong in my earlier comment - I think this code is still required